setup.py

```diff
@@ -4,10 +4,11 @@
 # Get the long description from the README file
 long_description = (Path(__file__).parent / "README.md").read_text(encoding="utf-8")
 
+# TODO how do we deal with different sub-packages here?
 setup(
-    name="bioimageio.core",
+    name="bioimageio.weight_converter",
```

Comment:
I suppose we should think about what bioimageio packages we want to end up with and what functionality we can maybe group together... but we can totally go ahead and have thie bioimageio.weight_converter package for now and change name/content later if need be...

Reply:
> I suppose we should think about what bioimageio packages we want to end up with and what functionality we can maybe group together... but we can totally go ahead and have thie bioimageio.weight_converter package for now and change name/content later if need be...

I agree, we need some proper discussion on how to structure this repo, see https://github.com/bioimage-io/python-bioimage-io/issues/74. But moving/renaming the `weight_converter` things is easy if we decide to, so I will go ahead and merge.